Week 1/Homework/Calculator.java

```diff
@@ -0,0 +1,31 @@
+public class Calculator{
```

Comment:
Warrant on the side of giving code more breathing room. It makes code easier to read.

```java
public class Calculator {
   if (true) {
    
   }
}
```